bcda/main_test.go

```diff
@@ -45,102 +59,139 @@ func (s *MainTestSuite) TestSetup() {
 func (s *MainTestSuite) TestAutoMigrate() {
 	// Plenty of other tests will rely on this happening
 	// Other tests run these lines so as long as this doesn't error it sb fine
-	autoMigrate()
+	args := []string{"bcda", "sql-migrate"}
+	err := s.testApp.Run(args)
+	assert.Nil(s.T(), err)
 }
 
 func (s *MainTestSuite) TestCreateACO() {
+
+	// init
 	db := database.GetGORMDbConnection()
-	//args := []string{CreateACO, "--name", "TEST_ACO"}
-	//err := s.testApp.Run(args)
-	//assert.Nil(s.T(), err)
-	//s.SetupAuthBackend()
+	s.SetupAuthBackend()
+
+	// set up the test app writer (to redirect CLI responses from stdout to a byte buffer)
+	buf := new(bytes.Buffer)
+	s.testApp.Writer = buf
```

Comment:
This is awesome. I didn't know about the `fmt.Fprint*()` functions allowing you to set the writer!